portality/lib/dataobj.py

```diff
@@ -759,7 +759,7 @@ def _get_list(self, path, coerce=None, by_reference=True, allow_coerce_failure=T
                 return deepcopy(val)
 
     def _set_single(self, path, val, coerce=None, allow_coerce_failure=False, allowed_values=None, allowed_range=None,
-                    allow_none=True, ignore_none=False):
+                    allow_none=True, ignore_none=False, additional_fields = None):
```

Comment:
This required at the moment as `additional_fields` is passed to the method at line 565. This can revisited afer thinking about implementing your suggestion to use `unicode_ascii_folded`

Reply:
@RK206 still  need to roll back this change on dataobj